manifests/plugin/abrt/params.pp

```diff
@@ -0,0 +1,13 @@
+# Default parameters for the Pulp smart proxy plugin
+class foreman_proxy::plugin::abrt::params {
+  $enabled                 = true
+  $group                   = undef
+  $abrt_send_log_file      = '/var/log/foreman-proxy/abrt-send.log'
+  $spooldir                = '/var/spool/foreman-proxy-abrt'
+  $aggregate_reports       = true
+  $send_period             = 600
+  $faf_server_url          = undef
+  $faf_server_ssl_noverify = true
+  $faf_server_ssl_cert     = "${::settings::ssldir}/certs/${::clientcert}.pem"
```

Comment:
::settings is a top-level hash provided by Puppet of the settings of the Puppet instance compiling the catalog.  I think it might be best to have the default empty as @mmilata said, else use one of the 'foreman' SSL class parameters as a default?